### DISCUSSION TITLE
gitlab pipeline failed in 0.50.0

### DISCUSSION THREAD

**Discussion by dmikots** [pointfreeco/swift-composable-architecture#1878] [Q&A] [answered]
<img width="896" alt="image" src="https://user-images.githubusercontent.com/57399131/215160765-cda4b74a-2251-462f-89a4-a008813d0697.png">
Pipeline tests in gitlab crash after 0.50.0 update

**Comment by stephencelis** [comment, marked as answer]
@dmikots There doesn't appear to be enough information for us to troubleshoot this issue. Can you explain more? Is this GitLab pipeline running tests against a TCA app of yours? Do the tests run fine for you locally?

**Reply by Alex0Klain** [reply]
@stephencelis For some reason, after release 0.50 CI stopped loading the TCA library with the error described above. We use such dependency versions in SPM:
```
.package(url: "https://github.com/pointfreeco/combine-schedulers", from: "0.5.0"),
.package(url: "https://github.com/pointfreeco/swift-tagged", from: "0.6.0"),
.package(
    url: "https://github.com/pointfreeco/swift-case-paths.git",
    from: "0.4.0"
  ),
  .package(
    url: "https://github.com/pointfreeco/swift-composable-architecture.git",
    from: "0.42.0"
  ),
  .package(url: "https://github.com/pointfreeco/swift-parsing", .upToNextMajor(from: "0.4.0")),
```
Maybe you know what could be the reason for such an error :\

**Reply by dmikots** [reply]
Same problem

**Reply by stephencelis** [reply]
One thing that stands out is the trailing `.git` in the repo names. I've seen SPM have issues when you include a package with `.git` in the repo URL, but another dependency depends on it without the `.git`. Can you change `swift-case-paths.git` to `swift-case-paths` and `swift-composable-architecture.git` to `swift-composable-architecture` and try again?

Also, can either of you attach a project that reproduces the problem? Maybe it boils down to putting a similar `Package.swift` file in a new project?

The `gh-pages` branch in the error may signal something about repo/branch size, which would be a troubling Xcode bug, but maybe we can work around it.

**Reply by stephencelis** [reply]
(Also, if you have, or your company has, a paid GitLab account, maybe they would be able to help diagnose the problem?)

**Reply by Alex0Klain** [reply]
@stephencelis Thanks for the help. The problem turned out to be in CI Runner, something broke in the cache in the package link. The complete removal of the runner cache helped
